src/lib/common/Kconfig

```diff
@@ -310,6 +310,10 @@ config SOL_BUS
 	default n
 
 menu "Hardware Options"
+config USE_AIO
+	bool "AIO Support"
```

Comment:
Is AIO the fabled POSIX Asynchronous IO? Might be a good idea to give this a more descriptive string (`bool "Analog I/O (AIO) support"`).